### PR TITLE
Module base_module_doc_rst migrated from version 6.1

### PR DESCRIPTION
**Pull request by sysadminmatmoz**
I use it as a tool for the relationship graph mostly, but it could be handy for someone else too.

Fresh PR since I messed up my previous git repo.

Added Pedro's change requests in README (todo) and the fact, that it performs it's actions only on actually installed modules in description.

The image gets presented only for the modules that are really currently installed (not just present as installable) in the same database where you run the creation button. The RST is the sphinx RST (as defined in description) - **the output file is a gzip archive** containing index.rst, module.rst and relations graph.

relation_graph

and sphinx output after make html:
sphinx1

### CODE REVIEW

**Comment by sysadminmatmoz** [comment]
I don't get the coveralls stats; if I look at only my module, the coverage increases.

**Comment by pedrobaeza** [comment]
You are adding code without tests, that's why the coverage decreases.

:+1: adding the known issues I commented in the other PR

**Comment by sysadminmatmoz** [comment]
My CLA is there for quite some time! Even from before I became an OCA member.

**Comment by StefanRijnhart** [comment]
Please rebase instead of merging the target branch. If you rebase now, you should get rid of the merge commits. It's some of the commits included in your merges that triggered the CLA warning.

**Comment by sysadminmatmoz** [comment]
Done. Thanks.

**Comment by sysadminmatmoz** [comment]
Finally all the checks are ok :smile:

File: base_module_doc_rst/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Generate Docs of Modules",
+    "version": "1.0",
```
Comment (StefanRijnhart) [inline base_module_doc_rst/__openerp__.py]: Please apply the new versioning convention and change to 8.0.1.0.0

File: base_module_doc_rst/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "category": "Tools",
+    "summary": "Modules Technical Guides in RST and Relationship Graphs",
+    "website": "https://odoo-community.org/",
+    "author": "OpenERP SA,Odoo Community Association (OCA)",
+    "contributors": [
```
Comment (StefanRijnhart) [inline base_module_doc_rst/__openerp__.py:29]: I don't think this is a valid key here. Contributors belong in the README.rst where you already put them.

File: base_module_doc_rst/wizard/tech_rst_guide.py [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+            "",
+            """
+            .. tip:: This module is part of the OpenERP software,
+            the leading Open Source
+            """,
+            """  enterprise management system. If you want to discover OpenERP,
+            check our
+            """,
+            """  `screencasts <http://openerp.tv>`_ or download """,
+            """  `OpenERP <http://openerp.com>`_ directly.""",
```
Comment (StefanRijnhart) [inline base_module_doc_rst/wizard/tech_rst_guide.py]: Maybe remove this old promotion bit.

**Comment by StefanRijnhart** [comment]
Just some nags but generally good, thanks!

**Comment by sysadminmatmoz** [comment]
Ok, I cleaned it further.
Regarding the contributors part: only in the server-tools there are 4 other modules with a openerp.py file set like that, but there are even a lot more of them in the other OCA repositories, and I took them as an example for a module, that is committed by someone that is not the original author.

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**